### DISCUSSION TITLE
How do I create a new cart with the frontend API when I get this error?

### DISCUSSION THREAD

**Discussion by gsumo** [swellstores/community#189] [Help] [answered]
{"error":{"code":"CART_EXPIRED","message":"Checkout session needs to be restarted","params":"DOMAIN"}}

What is the frontend JS code to restart a checkout?

**Comment by sags95** [comment, marked as answer]
Hey @gsumo, there was a short period where the frontend API was throwing this error. It's since been resolved however.
